Index: debian-groovy2/gradle/docs.gradle
===================================================================
--- debian-groovy2.orig/gradle/docs.gradle	2015-06-06 12:53:27.858164065 +0530
+++ debian-groovy2/gradle/docs.gradle	2015-06-06 13:01:55.155699212 +0530
@@ -6,17 +6,6 @@
     ext.title = "Groovy ${groovyVersion}"
 }
 
-if (JavaVersion.current().java7Compatible) {
-    task assembleAsciidoc(type: Copy, dependsOn: 'asciidocAll') {
-        subprojects {
-            from project.asciidoctor
-        }
-        into "$buildDir/asciidoc"
-    }
-
-    task asciidocAll(dependsOn: 'asciidoctor')
-}
-
 def javadocSpec = {
     maxMemory = javaDoc_mx
     project.configure(options) {
@@ -189,4 +178,4 @@
             options.addStringOption('Xdoclint:none', '-quiet')
         }
     }
-}
\ No newline at end of file
+}
